index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
+                            <h4>Sri Jaladi</h4>
+                            <h5>Teaching Assistant</h5>
+                            <h5>OH: Thurs 6 &ndash; 8pm </h5>
+                            <h5>Location: Zoom (link on Canvas calendar)</h5>
+                        </div>
+                        <div class="col-md-2 column">
+                            <img class="img-thumbnail square-image" src="images/daniel_shin.jpg" alt="Daniel Shin"
+                                width="160" height="160">
+                            <h4>Daniel Shin</h4>
+                            <h5>Teaching Assistant</h5>
+                            <h5>OH: Tues 5 &ndash; 7pm </h5>
+                            <h5>Location: Zoom (link on Canvas calendar) </h5>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+
+            <div class="hrline">
+                <hr />
+            </div> <br>
+            <h3 id="topics">Timeline</h3>
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th style="width: 12%">Date</th>
+                        <th style="width: 23%">Lecture</th>
+                        <th style="width: 15%">Deadlines</th>
+                        <th style="width: 50%">Notes</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr class="active">
+                        <td><b>Week 1</b><br />
+                            Wed, April 2
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Course intro + MDPs</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 1</b><br />
+                            Fri, April 4
+                        </td>
+                        <td><i><span class="label label-lecture text-base">TA Session</span></i>
+                            <a">Pytorch Tutorial</a>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+                            <ul>
+                                <li> Logistics: 1:30 - 2:50 pm in Gates B01</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr class="active">
+                        <td><b>Week 1</b><br />
+                            Fri, April 4
+                        </td>
+                        <td><i><span class="label label-lecture text-base">Lecture</span></i>
+                            <a">Imitation Learning</a>
+                        </td>
+                        <td>
+                            HW1 Out (IL)
```

Comment:
for all of the homework rows, could we leave it as "Homework 1 out" instead of "HW1 out (IL)"? I don't think we need to list the topics, and acronyms like "IL" might be confusing.